x-pack/plugins/security_solution/common/endpoint/types/actions.ts

```diff
@@ -44,10 +44,13 @@ export interface HostIsolationResponse {
   action: string;
 }
 
-export interface PendingActionsResponse {
+export interface HostPendingActions {
```

Comment:
listen, I really really really dislike nitpicks over naming.

But I recall a prior decision to eliminate "Host" from our team vocabulary.  

`EndpointPendingAction` is probably OK, since the results are filtered specifically to be actions that are only sent to the Endpoint process itself. `AgentPendingAction` is probably also OK, since these are "agent actions", and the struct is keyed/identified by `agent_id`. And the endpoint-filtering could change at any point and the struct wouldn't need to, so it could _technically_ contain actions for any destination through agent.

Reply:
Yeah, naming is not a strong skill of mine 😄 
I will change it to `EndpointPendingActions`